### ISSUE TITLE
Label com.joyrex2001.kubedock.runas-user not working

### ISSUE THREAD

**Issue by brutl1** [joyrex2001/kubedock#41]
We want to start a testcontainer with the label `com.joyrex2001.kubedock.runas-user` 

```
public class KafkaTestResource extends KafkaCompanionResource {

    @Override
    public void init(Map<String, String> initArgs) {
        if (this.kafkaCompanion == null) {
            super.init(initArgs);
            this.kafka.withLabel("com.joyrex2001.kubedock.runas-user", "0");
            this.kafka.withLabel("com.joyrex2001.kubedock.request-cpu", "400m,1");
```

The label does not change the `runasUser` Setting in the created Pod. 
See the following output of the kubedock log where the labels are visible but kubedock starts the pod with the user defined in image: 

```
I0707 08:57:12.146529       1 util.go:78] Request Body: {"Hostname":null,"Domainname":null,"User":null,"AttachStdin":null,"AttachStdout":null,"AttachStderr":null,"PortSpecs":null,"Tty":null,"OpenStdin":null,"StdinOnce":null,"Env":["LOG_DIR=/tmp"],"Cmd":["sh","-c","while [ ! -f /testcontainers_start.sh ]; do sleep 0.1; done; /testcontainers_start.sh"],"Healthcheck":null,"ArgsEscaped":null,"Entrypoint":null,"Image":"quay.io/strimzi-test-container/test-container:0.100.0-kafka-3.1.0","Volumes":{},"WorkingDir":null,"MacAddress":null,"OnBuild":null,"NetworkDisabled":null,"ExposedPorts":{"9092/tcp":{}},"StopSignal":null,"StopTimeout":null,"HostConfig":{"Binds":[],"BlkioWeight":null,"BlkioWeightDevice":null,"BlkioDeviceReadBps":null,"BlkioDeviceWriteBps":null,"BlkioDeviceReadIOps":null,"BlkioDeviceWriteIOps":null,"MemorySwappiness":null,"NanoCpus":null,"CapAdd":null,"CapDrop":null,"ContainerIDFile":null,"CpuPeriod":null,"CpuRealtimePeriod":null,"CpuRealtimeRuntime":null,"CpuShares":null,"CpuQuota":null,"CpusetCpus":null,"CpusetMems":null,"Devices":null,"DeviceCgroupRules":null,"DeviceRequests":null,"DiskQuota":null,"Dns":null,"DnsOptions":null,"DnsSearch":null,"ExtraHosts":[],"GroupAdd":null,"IpcMode":null,"Cgroup":null,"Links":[],"LogConfig":null,"LxcConf":null,"Memory":null,"MemorySwap":null,"MemoryReservation":null,"KernelMemory":null,"NetworkMode":"95cde802aa72b1b9dd5955feb45e93ad6ca3bb2ef27388caa24d85c23a6d13f7","OomKillDisable":null,"Init":null,"AutoRemove":null,"OomScoreAdj":null,"PortBindings":{"9092/tcp":[{"HostIp":"","HostPort":""}]},"Privileged":null,"PublishAllPorts":null,"ReadonlyRootfs":null,"RestartPolicy":null,"Ulimits":null,"CpuCount":null,"CpuPercent":null,"IOMaximumIOps":null,"IOMaximumBandwidth":null,"VolumesFrom":[],"Mounts":null,"PidMode":null,"Isolation":null,"SecurityOpt":null,"StorageOpt":null,"CgroupParent":null,"VolumeDriver":null,"ShmSize":null,"PidsLimit":null,"Runtime":null,"Tmpfs":null,"UTSMode":null,"UsernsMode":null,"Sysctls":null,"ConsoleSize":null,"CgroupnsMode":null},"Labels":{"com.joyrex2001.kubedock.runas-user":"0","org.testcontainers":"true","org.testcontainers.lang":"java","com.joyrex2001.kubedock.request-cpu":"400m,1","org.testcontainers.version":"1.17.6","org.testcontainers.sessionId":"4b7fc7b7-b48b-4059-866c-713f470b8861"},"Shell":null,"NetworkingConfig":{"EndpointsConfig":{"95cde802aa72b1b9dd5955feb45e93ad6ca3bb2ef27388caa24d85c23a6d13f7":{"IPAMConfig":null,"Links":null,"Aliases":["tc-qkimr7Zz"],"NetworkID":null,"EndpointID":null,"Gateway":null,"IPAddress":null,"IPPrefixLen":null,"IPv6Gateway":null,"GlobalIPv6Address":null,"GlobalIPv6PrefixLen":null,"MacAddress":null}}}}
I0707 08:57:12.146797       1 util.go:101] Response Body: {"Id":"d9b8a736e1ddb306f38e172f517b4cf0482d07709d721641d2d2bfb253a1fd29"}
[GIN] 2023/07/07 - 08:57:12 | 201 |     393.293Âµs |      10.131.1.9 | POST     "/containers/create"
I0707 08:57:12.211618       1 util.go:77] Request Headers: http.Header{"Accept":[]string{"application/json"}, "Accept-Encoding":[]string{"gzip, x-gzip, deflate"}, "Connection":[]string{"keep-alive"}, "Content-Type":[]string{"application/json"}, "User-Agent":[]string{"Apache-HttpClient/5.0.3 (Java/17.0.5)"}, "X-Tc-Sid":[]string{"4b7fc7b7-b48b-4059-866c-713f470b8861"}}
I0707 08:57:12.211645       1 util.go:78] Request Body: 
W0707 08:57:12.211698       1 container.go:174] user not set, will run as user defined in image
```

I've quickly searched through the repository and found the `const` for the labels [here](https://github.com/joyrex2001/kubedock/blob/master/internal/model/types/container.go#L57) but there is no const for the label `com.joyrex2001.kubedock.runas-user`. 

We do not want to run any Pod with the UID 0 so the flag `--runas-user` is no solution.  

Am I doing something wrong or is the implementation of the label missing?

**Comment by joyrex2001** [comment]
Good find! This was indeed not implemented in #5, added it.

**Comment by brutl1** [comment]
Nice, thanks for the quick implementation!
